src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
```

```suggestion
.stars {
  display: flex;
```
и задай всем блокам один общий класс stars